tests/forms_tests/widget_tests/test_select.py

```diff
@@ -111,6 +111,28 @@ def test_constructor_attrs(self):
             ),
         )
 
+    def test_constructor_option_attrs(self):
```

Comment:
I believe we need an analogous test for radioselect, and even more so that radioselect has the `option_inherits_attrs` set to True, so we'd need tests for `attrs` and `option_attrs` being passed to ensure that `option_attrs` takes precedence.